### PR TITLE
[vulkan-headers, vulkan-hpp ,vulkan-memory-allocator] Update to the latestion version

### PR DESCRIPTION
**Pull request by chrismile**
**Describe the pull request**

- #### What does your PR fix?  
  This PR updates vulkan-headers, vulkan-hpp and vulkan-memory-allocator to more recent versions (1.2.184 for vulkan-headers and vulkan-hpp).

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All, no.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes, except "Make separate Pull Requests per port". I decided to include all three updates in one PR, as they all concern Vulkan ports and an update with respect to Vulkan 1.2.184.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes.

### CODE REVIEW

File: ports/ashes/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ashes",
-  "version-string": "2021-02-09",
+  "version-string": "2021-06-18",
```
Comment (JonLiu1993) [inline ports/ashes/vcpkg.json:3]:
```suggestion
  "version-date": "2021-06-18",
```

File: ports/vulkan-headers/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vulkan-headers",
-  "version-string": "1.2.157",
+  "version-string": "1.2.184",
```
Comment (JonLiu1993) [inline ports/vulkan-headers/vcpkg.json:3]:
```suggestion
  "version-semver": "1.2.184",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@chrismile ,Please run the command "./vcpkg x-add-version --all --overwrite-version" and commit the change again.

**Comment by chrismile** [comment]
Thanks a lot for your feedback! All requested changes should now be integrated in the PR.

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**